change/@ni-nimble-components-c5cf9aa2-8b3b-430e-99db-7c820f097578.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Mapping and icon table columns",
```

Comment:
As a note for future PRs, this PR ended up being really big, which makes it a lot more difficult as a reviewer to provide a thorough review. I think it would have been helpful to split out the two different column types into two different PRs and possibly have tried to get creative in additional ways to split things out into smaller pieces.

Reply:
Yeah, I'll try to watch out for that in the future.